hr_timesheet_activity_begin_end_readonly_duration/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# © 2023 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+{
+    "name": "Timesheet Activities - Begin/End Hours - Readonly Duration",
+    "summary": """
+        Change the readonly attribute of field unit_amount to true
+        to force the Begin Hour and End Hour to be added
+    """,
+    "author": "Solvos",
+    "license": "LGPL-3",
```

Comment:
This module depends on an OCA addon, so we should this license instead
```suggestion
    "license": "AGPL-3",
```